### PR TITLE
Gyroflow minimum settings

### PR DESCRIPTION
**Pull request by Garban5**
Disables all blackblox fields which are not necessary to use [Gyroflow](https://gyroflow.xyz). Only gyro data will be recorded at the lowest logging rate possible. This allows for smaller blackbox files, longer blackbox recordings and lower CPU overhead.

The space gain is substantial. For every 10 seconds of flight, with the minimum configuration the space occupied is approximately 32.5kB and for a typical blackbox configuration (debug mode in Gyro_scaled and a rate of 2kHz) the occupation is 574.7kB
These values are approximate and should be taken as an example. Every drone and every flight are different.

### CODE REVIEW

**Review by limonspb — DISMISSED** [review]

File: presets/4.3/other/gyroflow_min_blackbox.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#$ TITLE: Gyroflow minimum settings
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: OTHER
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: gyroflow, blackbox, log, DDC, Acrobaix, Garban 
+#$ AUTHOR: Garban
+#$ DESCRIPTION: Minimum requirements settings for Gyroflow.
```
Comment (KarateBrot) [inline presets/4.3/other/gyroflow_min_blackbox.txt:7]:
```suggestion
#$ DESCRIPTION:
#$ DESCRIPTION: <img src="https://github.com/gyroflow/gyroflow/raw/master/resources/logo_black.svg" width="250px" style="margin-left: auto; margin-right: auto; display: block;"/>
#$ DESCRIPTION:
#$ DESCRIPTION: Description
#$ DESCRIPTION: ------------
#$ DESCRIPTION: Disables all blackblox fields which are not necessary to use [Gyroflow](https://gyroflow.xyz). Only gyro data will be recorded at the lowest logging rate possible. This allows for smaller blackbox files, longer blackbox recordings and smaller CPU overhead.

#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/223
```

File: presets/4.3/other/gyroflow_min_blackbox.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#$ TITLE: Gyroflow minimum settings
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: OTHER
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: gyroflow, blackbox, log, DDC, Acrobaix, Garban 
+#$ AUTHOR: Garban
```
Comment (KarateBrot) [inline presets/4.3/other/gyroflow_min_blackbox.txt:6]:
```suggestion
#$ AUTHOR: Garban

#$ PARSER: MARKED

```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

File: presets/4.3/other/gyroflow_min_blackbox.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#$ TITLE: Gyroflow minimum settings
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: OTHER
+#$ STATUS: COMMUNITY
```
Comment (sugaarK) [inline presets/4.3/other/gyroflow_min_blackbox.txt:4]: should be experimental until more community testing has been vitrified

Comment (Garban5) [reply presets/4.3/other/gyroflow_min_blackbox.txt]: ok, done!

**Review by Garban5 — COMMENTED** [review]
ok

**Review by sugaarK — APPROVED** [review]
Looks good to me now.

**Comment by sugaarK** [comment]
> Disables Blackblox information that is not needed to use Gyroflow. This allows save more records in the Blackbox.

Any testing to indicate how much more logging time this gains just out of interest?

**Comment by Garban5** [comment]
> The space gain is substantial. For every 10 seconds of flight, with the minimum configuration the space occupied is approximately 32.5kbytes and for a typical blackbox configuration (debug mode in Gyro_scaled and a rate of 2kHz) the occupation is 574.7k.
> These values are approximate and should be taken as an example. Every drone and every flight are different.

**Review by Garban5 — COMMENTED** [review]
ok

**Comment by Garban5** [comment]
I have had some problem with git to merge the commits and I can't merge these. I hope I haven't broken anything.
Sorry

**Review by KarateBrot — DISMISSED** [review]

**Comment by limonspb** [comment]
> @Garban5 All good :)
> 
> @limonspb Do you have godmode to squash?

can't squash-merge here :/
image

@Garban5 could you please squash commits in one commit?

**Comment by Garban5** [comment]
Sorry @sugaarK  @limonspb  @KarateBrot finally I can't merge, and open a clean pull request and close this.

**Pull request closed without merging**